### ISSUE TITLE
<GVT><non-EN> Hardcode string "Loading" shows in loading component

### ISSUE THREAD

**Issue by reeteshsingh93** [carbon-design-system/carbon#4194]
## Environment

> Operating system

Mac OS MOjave

> Browser

FIrefox
> Automated testing tool and ruleset
N.A
> Assistive technology used to verify

## Detailed description
> What version of the Carbon Design System are you using?
    "carbon-components": "9.57.0",
    "carbon-components-react": "6.115.1",
> What did you expect to happen?
 Should display localized text as per the borwser settings
> What happened instead?
display text in english
> What WCAG 2.1 checkpoint does the issue violate?

## Steps to reproduce the issue

1. Step one

Render <Loading> component on any page 
2. Step two
Change the browser local to chinese 
3. Step three
 hover over the component text will appear in english.
4. etc.

> Please create a reduced test case in CodeSandbox
>
> - Style and vanilla JS:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/components/examples/codesandbox
> - React:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox

## Additional information

- Screenshots or code

<img width="391" alt="image" src="https://user-images.githubusercontent.com/46310374/66105998-77f9c680-e5da-11e9-8746-42e43ab4ec39.png">

- Notes

**Comment by asudoh** [comment]
Hi 👋 thank you for reporting! Feel free to create a backport PR of: #2955

**Comment by reeteshsingh93** [comment]
I don't have access to org for raising a PR: 

![image](https://user-images.githubusercontent.com/46310374/67275636-a9322c00-f4e0-11e9-8d9b-773c1f7204be.png)

**Comment by reeteshsingh93** [comment]
@asudoh @emyarod Please suggest..

**Comment by emyarod** [comment]
@reeteshsingh93 there is already an open PR https://github.com/carbon-design-system/carbon-components-react/pull/2421 but is it possible your credentials or your remote are incorrect?

**Comment by asudoh** [comment]
@reeteshsingh93 Our process for creating PR is creating a PR from a fork, instead of directory to this repo. Thanks!

**Comment by reeteshsingh93** [comment]
@emyarod Are you plainning to merge that PR? and will it resolve issue in "carbon-components-react": "6.115.1" after it get merged?

**Comment by reeteshsingh93** [comment]
@asudoh Ok will raise it that way if required.

**Comment by emyarod** [comment]
it will be in [v6.116.2](https://github.com/carbon-design-system/carbon-components-react/releases/tag/v6.116.2)

**Comment by reeteshsingh93** [comment]
@emyarod Thanks for your support.
